srcpkgs/atom-bin/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'atom-bin'
+pkgname=atom-bin
```

Comment:
What's wrong with the atom package?

Reply:
Nothing wrong just not up to date.

Reply:
Then please try to update that one instead?

Reply:
This was not resolved, but I can let it slide....

Reply:
i'd rather not let it slide

Reply:
In that case: I think this PR can be closed, because the correct course is for us to build atom.